recipes/_install.rb

```diff
@@ -15,7 +15,7 @@ def nsp
 node.default['splunk']['package']['file_name'] = "#{nsp['name']}#{nsp['file_suffix']}"
 node.default['splunk']['package']['url'] =
   "#{nsp['base_url']}/#{nsp['download_group']}/releases/#{nsp['version']}/#{nsp['platform']}/#{nsp['file_name']}"
-node.default['splunk']['home'] = CernerSplunk.splunk_home(node['platform_family'], node['kernel']['machine'], nsp['base_name'])
+node.default['splunk']['home'] = CernerSplunk.splunk_home(node['platform_family'], node['kernel']['machine'], CernerSplunk.installed_package_name(node['platform_family'], node['splunk']['package']['base_name']))
```

Comment:
When I test this on the f_win2012r2 vagrant box, the forwarder is still installed under `C:\Program Files\splunkforwarder`, so the existing values seem correct.  Does this needs to flex based on windows version?  I'm trying to get a newer version of windows for the vagrant testing, but having some issues finding a working vagrant box.

Reply:
Nevermind..forgot we were setting the INSTALLDIR flag during the windows_package install.  If I don't set the INSTALLDIR, the default appears to actually be SplunkUniversalForwarder as opposed to just UniversalForwarder.  So what does changing from `splunkforwarder` to `UniversalForwarder` for the install dir actually do?

Reply:
It does seem like INSTALLDIR being set would prevent this from being an issue, but we're seeing all of our nodes install to C:\program files\UniversalForwarder instead of splunkforwarder. When chef runs for a second time, it bombs on the `windows_package` resource below because `manifest_missing` goes to check C:\program files\splunkforwarder for the manifest file.

I may need to investigate why it's installing to a that directory instead of the one specified in INSTALLDIR

Reply:
Okay, it looks like this may not be the fix I'm after, nor would it provide any meaningful benefit right now. I'll close this out.